### PR TITLE
Rethemed KYC modal to match new dark theme

### PR DESCRIPTION
**Pull request by itayplav**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [x] Tested on small Android
- [ ] n/a
image

### CODE REVIEW

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

**Comment by swansontec** [comment]
@itayplav, please rebase this onto develop & fix the conflicts. Then pass the task back to @thehobbit85 for review.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

File: src/components/modals/SwapVerifyTermsModal.js [hunk 1]
```diff
@@ -35,38 +65,65 @@ export async function swapVerifyTerms(swapConfig: EdgeSwapConfig, links: Array<{
...
+const getStyles = cacheStyles(theme => ({
+  linkContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between'
+  },
+  linkText: {
+    color: theme.iconTappable,
+    flexShrink: 1,
+    fontFamily: theme.fontFaceDefault,
+    fontSize: theme.rem(0.84),
```
[the remark this answers is not in the document]
Comment (itayplav) [reply src/components/modals/SwapVerifyTermsModal.js:117]: It references the `small` font size in [textStyles.js](https://github.com/EdgeApp/edge-react-gui/blob/13b995842ad952740304372d174a733d65c03640/src/styles/common/textStyles.js#L44-L44)

File: src/components/modals/SwapVerifyTermsModal.js [hunk 2]
```diff
@@ -35,38 +65,65 @@ export async function swapVerifyTerms(swapConfig: EdgeSwapConfig, links: Array<{
...
+      <View style={styles.titleContainer}>
+        <Image style={styles.titleImage} source={swapPluginIcons[pluginId]} resizeMode="contain" />
+        <ModalTitle>{displayName}</ModalTitle>
+      </View>
+      <ModalMessage>{s.strings.swap_terms_statement}</ModalMessage>
+      <MainButton label={s.strings.swap_terms_accept_button} marginRem={0.5} onPress={() => bridge.resolve(true)} />
+      <MainButton label={s.strings.swap_terms_reject_button} marginRem={0.5} type="secondary" onPress={() => bridge.resolve(false)} />
+      <View style={styles.linkContainer}>
+        {termsUri == null ? null : (
+          <Text style={styles.linkText} onPress={() => Linking.openURL(termsUri)}>
```
[the remark this answers is not in the document]
Comment (itayplav) [reply src/components/modals/SwapVerifyTermsModal.js:89]: @swansontec discourages the use of EdgeText

File: src/actions/CryptoExchangeActions.js [hunk 3]
```diff
@@ -326,23 +326,20 @@ export const shiftCryptoCurrency = (swapInfo: GuiSwapInfo) => async (dispatch: D
   const { quote, request } = swapInfo
   const { pluginId, toNativeAmount } = quote
   const { fromWallet, toWallet, fromCurrencyCode, toCurrencyCode } = request
-
```
Comment (swansontec) [inline src/actions/CryptoExchangeActions.js:329]: Please put back the blank line after imports.

File: src/actions/CryptoExchangeActions.js [hunk 4]
```diff
@@ -326,23 +326,20 @@ export const shiftCryptoCurrency = (swapInfo: GuiSwapInfo) => async (dispatch: D
...
-    let category: string
-    let name: string
-    if (pluginId === 'transfer') {
-      category = sprintf('transfer:%s %s %s', fromCurrencyCode, s.strings.word_to_in_convert_from_to_string, toWallet.name)
-      name = toWallet.name || ''
-    } else {
-      category = sprintf('exchange:%s %s %s', fromCurrencyCode, s.strings.word_to_in_convert_from_to_string, toCurrencyCode)
-      name = si.displayName
-    }
+    // const
```
Comment (swansontec) [inline src/actions/CryptoExchangeActions.js:336]: Remove this extra const.

File: src/components/scenes/CryptoExchangeQuoteScene.js [hunk 5]
```diff
@@ -274,27 +183,16 @@ const getStyles = cacheStyles((theme: Theme) => ({
...
-      pending: state.cryptoExchange.shiftPendingTransaction,
-      toCurrencyIcon: state.cryptoExchange.toCurrencyIcon || '',
-      toDenomination: state.cryptoExchange.toWalletPrimaryInfo.displayDenomination.name,
-      toWalletCurrencyName
-    }
-  },
+  (state, { route: { params } }) => ({
+    account: state.core.account,
+    fromCurrencyIcon: state.cryptoExchange.fromCurrencyIcon ?? '',
+    fromDenomination: state.cryptoExchange.fromWalletPrimaryInfo.displayDenomination.name ?? '',
```
Comment (swansontec) [inline src/components/scenes/CryptoExchangeQuoteScene.js:189]: The `displayDenomination.name` isn't optional, so no need for `?? ''`

**Review by swansontec — CHANGES_REQUESTED** [review]
Three tiny changes.

File: src/components/scenes/CryptoExchangeQuoteScene.js (hunk 5 above)
Comment (swansontec) [reply src/components/scenes/CryptoExchangeQuoteScene.js:189]: The other `?? ''` are still needed, though. It's just this one line.

**Review by swansontec — APPROVED** [review]
I made the 3 tiny changes myself. Approved.

**Pull request merged**